src/androidTest/java/com/mixpanel/android/viewcrawler/ViewSnapshotTest.java

```diff
@@ -32,11 +31,11 @@ public void setUp() throws NoSuchMethodException {
 
         final List<PropertyDescription> props = new ArrayList<PropertyDescription>();
 
-        final Caller textGetter = new Caller(TextView.class, "getText", new Object[0], CharSequence.class);
+        final PropertyCaller textGetter = new PropertyCaller(TextView.class, "getText", new Object[0], CharSequence.class);
```

Comment:
I'd prefer to keep it PropertyCaller, as we have 2 types of change now. Naming it Caller makes it a bit confused, it implies Caller may get used by LayoutChangeVisitor as well. Do you prefer to call it Caller still?